docs/content/blog/authelia-traefik-setup-guide/index.md

```diff
@@ -78,19 +96,29 @@ services:
     secrets:
       - cloudflare_email
       - cloudflare_api_key
+    labels:
+      traefik.enable: 'true'
+      traefik.http.routers.dashboard.rule: 'Host(`traefik.{{< sitevar name="domain" nojs="example.com" >}})'
+      traefik.http.routers.dashboard.entrypoints: 'https'
+      traefik.http.routers.dashboard.middlewares: 'authelia@docker'
+      traefik.http.routers.dashboard.service: 'api@internal'
+      traefik.http.routers.dashboard.tls.certresolver: 'cloudflare'
+      traefik.http.middlewares.authelia.forwardAuth.address: '{{< sitevar name="tls" nojs="http" >}}://{{< sitevar name="host" nojs="authelia" >}}:{{< sitevar name="port" nojs="9091" >}}/api/authz/forward-auth'
+      traefik.http.middlewares.authelia.forwardAuth.trustForwardHeader: 'true'
+      traefik.http.middlewares.authelia.forwardAuth.authResponseHeaders: 'Remote-User,Remote-Groups,Remote-Name,Remote-Email'
```

Comment:
Again probably more a WDYT as opposed to a hard ask but I can see you moved this from the Authelia container to the Traefik container, shouldn't we define the Authelia middleware along with its own container?

Reply:
Correct, I want to make sure all the config actually works so there may be some artifacts of me working things around to get it working.